UKHomeOffice/cop-ui
# COP-10232: Hide "View in-progress forms" link behind flag

### Description
This shouldn't be shown in the live environment until such time as there are live React forms as the Form.io ones cannot be saved as drafts.

The consequence of this change is that the link will now ONLY show if the environment on which it is deployed has an AWS secret for `inProgressForms` to be set to `true` or `'true'` (or you have `REACT_APP_IN_PROGRESS_FORMS=true` in your local env file.)

### To test
The link for "View in-progress forms" on the Home Page should be shown on an environment where the flag is turned on... and not shown the flag is turned off (or not set).
